workflows/terraform_and_release.yml

```diff
@@ -0,0 +1,45 @@
+name: 'CICD'
+on:
+  push:
+  pull_request:
+jobs:
+  terraform:
+    name: 'Terraform'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout'
+        uses: actions/checkout@master
+
+      - uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 0.13.x
+
+      - run: terraform fmt
+      - run: terraform init
+
+      - name: 'Inject provider configs for validate command'
+        run: |
+            cat > providers.tf <<EOF
+            provider "aws" {
+              region = "us-east-2"
+            }
+            EOF
+      - run: terraform validate
+
+  release:
+   if: github.event_name == 'push'
```

Comment:
Because the `on` parts of the YAML (line 2) are empty, doesn't this mean that a release action is going to be pushed for every push to a topic or default branch?